docs/reference.rst

```diff
@@ -561,12 +561,40 @@ A valid RAUC manifest file must be named ``manifest.raucm``.
 
 **[handler] section**
 
+The ``handler`` section refers to the *full custom handler* that allows
+to fully replace the default RAUC update process.
```

Comment:
Add a reference to https://rauc.readthedocs.io/en/latest/using.html#full-custom-update.

Reply:
This hardly gave any additional info. I have added the link but also slightly extended the chapter that it refers to